Resources/Prototypes/Catalog/Fills/Lockers/medical.yml

```diff
@@ -138,14 +135,12 @@
     contents:
 #      - id: ClothingHeadHelmetVoidParamed # Frontier
 #      - id: ClothingOuterHardsuitVoidParamed # Frontier
-      - id: ClothingOuterCoatParamedicWB
-      - id: ClothingHeadHatParamedicsoft
-      - id: ClothingUniformJumpsuitParamedic
-      - id: ClothingUniformJumpskirtParamedic
+#      - id: ClothingOuterCoatParamedicWB
+#      - id: ClothingHeadHatParamedicsoft
+#      - id: ClothingUniformJumpsuitParamedic
+#      - id: ClothingUniformJumpskirtParamedic
       - id: ClothingEyesHudMedical
-      - id: ClothingHandsGlovesLatex
-      - id: ClothingHeadsetMedical
-      - id: ClothingMaskSterile
-      - id: MedkitFilled
-      - id: HandLabeler # Frontier
-      - id: ClothingBeltMedical # Frontier
+#      - id: ClothingHandsGlovesLatex
+#      - id: ClothingHeadsetMedical  # Frontier
+#      - id: ClothingMaskSterile
+      - id: ClothingBeltParamedicFilled # Frontier
```

Comment:
you need to leave the things like gloves and masks in because theres literally no way for them to get it.